app/src/main/java/com/readtracker/android/activities/ImportActivity.java

```diff
@@ -30,21 +35,23 @@
   private static final String TAG = ImportActivity.class.getSimpleName();
 
   @InjectView(R.id.current_folder_text) TextView currentFolderText;
+  @InjectView(R.id.button_common_dirs) TextView commonDirsButton;
   @InjectView(R.id.file_list) ListView fileList;
 
   private FileBrowserAdapter fileBrowseAdapter;
```

Comment:
```fileBrowseAdapter``` -> ```mFileBrowseAdapter``` to keep the naming conventions consistent?

Reply:
I've actually grown to dislike the Hungarian notation (even as a Hungarian 🤓), so I would like to retract my original (old) review comment on this